gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -132,7 +132,7 @@ local map_switch_delay = CreateConVar(
 
 ---
 -- @realm server
-CreateConVar(
+local identify_body_woconfirm = CreateConVar(
```

Comment:
best would be to move this line to a shared file and add the replicated convar flag. Global bools were added in the past when replication was broken, we're slowly trying to remove this from the codebase in recent updates.

Reply:
Mmm, while we're there, should I also remove the one for `ttt2_confirm_team` and move its `CreateConVar` call to `sv_corpse`? If GlobalVars are gonna be cleaned up and replaced with replicated ConVars, I figure we might as well move it to where it belongs.

There's also a reference to `ttt2_confirm_team` in `/lua/autorun/gs_crazyphysics.lua` that uses its `GetGlobal`, should we just store a copy of the ConVar object in that script as well?